README.md

```diff
@@ -48,39 +50,271 @@ class FooPresenter
+  formats :created_at,
+          with: 'DateTimeFormatter',
+          options: {
+            format: '%-d %B %Y at %H:%M'
+          }
+end
+```
+
+If needed, override those options where the formatted value is being rendered:
+
+```erb
+<% FooPresenter.present @foo do |foo_presenter, _foo| %>
+  <%= foo_presenter.created_at.format('%c').blank_value('–') %>
+<% end %>
+```
+
+#### Named formatters
+
+Optionally configure formatters with a name, for example in a Rails initializer:
+
+```ruby
+require 'chic'
+
+Chic.configure do |config|
+  config.formatters.merge!(
+    date_time: 'DateTimeFormatter'
+  )
+end
+```
+
+Allowing you to refer to those formatters by name instead of by class:
+
+```ruby
+class FooPresenter < Chic::Presenter
+  formats :created_at,
+          with: :date_time
+end
+```
+
+## Logging
+
+If a presenter class for an object you're trying to present can't be found, an entry at debug level will be made to the configured logger.
+
+You can configure the logger to be used:
+
+```ruby
+require 'chic'
+
+Chic.configure do |config|
+  config.logger = Logger.new($stdout)
+end
+```
+
+It may be beneficial to know about missing presenter classes sooner than later, in which case you can enable exceptions when it makes sense to do so – for example, a Rails application in any environment other than production:
+
+```ruby
+require 'chic'
+
+Chic.configure do |config|
+  config.raise_exceptions = Rails.env.production? == false
+end
+```
+
+## Conventions
+
+A few helpful conventions that have gone a long way to keep things maintainable.
+
+### Instantiate presenters in views only
+
+**Don't**
+
+```ruby
+# app/controllers/foo_controller.rb
+
+class FoosController < ApplicationController
+  def show
+    @foo = Foo.find(params[:id])
+    @foo_presenter = FooPresenter.new(@foo)
+  end
+end
+```
+
+```erb
+<!-- app/views/foos/_show.html.erb -->
+
+<%= link_to @foo_presenter.created_at, foo_path(@foo) %>
 ```
 
-And:
+**Do**
 
 ```ruby
-<% present_each @foo do |foo_presenter, _foo| %>
+# app/controllers/foo_controller.rb
+
+class FoosController < ApplicationController
+  def show
+    @foo = Foo.find(params[:id])
+  end
+end
+```
+
+```erb
+<!-- app/views/foos/_show.html.erb -->
+
+<% present @foo do |foo_presenter| %>
+  <%= link_to foo_presenter.created_at, foo_path(@foo) %>
+<% end %>
+```
+
+### Keep presenter instances scoped to only one view
+
+**Don't**
+
+```erb
+<% present @foo do |foo_presenter| %>
+  <!-- ... -->
+  <%= render partial: 'path/to/partial', locals: { foo: foo_presenter } %>
+<% end %>
+```
+
+**Do**
+
+```erb
+<% present @foo do |foo_presenter| %>
   <!-- ... -->
+  <%= render partial: 'path/to/partial', locals: { foo: @foo } %>
+<% end %>
+```
+
+### Use presenters and formatters for presentation only
+
+**Don't**
+
+```erb
+<% present_each @foos do |foo_presenter, foo| %>
+  <% if foo_presenter.created_at %>
+    <%= link_to foo_presenter.created_at, foo_path(foo_presenter) %>
```

Comment:
Just a little confused about why `foo_path(foo_presenter)` would even work. :confused: 

Would it delegate `.id` to `object` implicitly?

Reply:
You're right, bad example – it would only work if `id` was a formatted attribute, which is not guaranteed. This is a hangover from the days where presenters used to derive from `SimpleDelegator`.